test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -311,6 +416,38 @@ public async Task GivenIncrementalLoad_SameLastUpdated_DifferentVersions_Resourc
             Assert.Equal(GetLastUpdated("2002"), result.Resource.Meta.LastUpdated);
         }
 
+        [Fact]
+        public async Task GivenIncrementalLoad_SameLastUpdated_DifferentVersions_DifferentImports_ResourceExisting()
```

Comment:
Same question as above, should the response be checked to show a message about ndJson1 is being returned?

Reply:
The check is on line 437. It tells that one record was not loaded. That import contained only one record, hence we know that it was not loaded.